.github/workflows/ci-openapi.yml

```diff
@@ -137,3 +137,54 @@ jobs:
             <!--openapi-diff-workflow-comment-->
 
             No changes to OpenAPI specification found. See history of this comment for previous changes.
+
+  publish:
+    name: OpenAPI - Publish Unstable Spec
+    if:
+      - github.event_name != 'pull_request'
+      - contains(github.repository_owner, 'jellyfin')
```

Comment:
I don't think you are allowed to use an array here

Reply:
Aah yes you are correct, needs to be a YAML block.